fedimintd/src/lib.rs

```diff
@@ -1,12 +1,15 @@
 use bitcoin::Network;
-use fedimint_core::config::{ConfigGenParams, ModuleGenRegistry};
-use fedimint_core::module::DynModuleGen;
+use fedimint_core::config::ConfigGenParams;
 use fedimint_core::{Amount, Tiered};
-use fedimint_ln::LightningGen;
-use fedimint_mint::{MintGen, MintGenParams};
-use fedimint_wallet::{WalletGen, WalletGenParams};
+use fedimint_mint::MintGenParams;
+use fedimint_wallet::WalletGenParams;
 
-pub mod ui;
+mod ui;
+
+/// Module for creating `distributetgen` binary with custom modules
+pub mod distributed_gen;
+/// Module for creating `fedimintd` binary with custom modules
+pub mod fedimintd;
 
 /// Generates the configuration for the modules configured in the server binary
 pub fn configure_modules(
```

Comment:
Should this module-specific code also be in the `bin`?

Reply:
The "fork"-binary will only link module kinds with the rest of `fedimintd` code. Everything else should stay in the general code.

These should come from the config/UI eventually.